### ISSUE TITLE
Issue with Setup.py

### ISSUE THREAD

**Issue by pinballsavvy** [zoony1337/TikTok-Shares-Botter#3]
When I run the setup program I get the text 

"'pip' is not recognized as an internal or external command,
operable program or batch file."

This makes it so when I try to open the Main.py program is opens and close to quickly to see anything.  I don't know if these things have anything to do with each other but if you could help me out that would be great!

**Comment by nikokl** [comment]
reinstall python and make sure whenever you first launch the installer you check the box that says something about "PATH". i dont remember exactly what it said but that fixed it for me.

**Comment by ViniBoi604** [comment]
can someone please help me, i am doing the share botter and everything but the shares are not showing up anymore. it was working fine an hour ago, now its not

**Comment by pinballsavvy** [comment]
Yeah unfortunately TikTok must have patched this due to so many people doing it.

**Comment by zoony1337** [comment]
It was patched, but I updated the code and it works again.

**Comment by henmasgat** [comment]
I reninstalled it and clicked on Path but main py keeps on crashing
